definitions/ext-cdn_generic/synthetic_script.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
this is probably not intentional?

Reply:
The synthetic script is intentional. It creates a new CDN event by looking at SyntheticRequests where the URL contains "cdn". Simple approach and can easily be made more robust and extended to include Browser and Mobile requests. 

Since users wouldn't have CDN events without this script, we've included it here.

Reply:
It may be worth turning this into an Observability Pack at somepoint, and convert the synthetic into a nerd-graph call that creates the synthetic in the customer's account.